vello_hybrid/README.md

```diff
@@ -0,0 +1,40 @@
+# Vello Hybrid Crates
+
```

Comment:
It would be nice to have an introduction sentence here, explaining what this folder actually is. We currently explain reasonably well what it contains, but not e.g. why it exists

Something like:

```suggestion

We're developing a new implementation of Vello, which is:
- Compatible with a wider set of devices
- Removes some performance cliffs
- Works in more memory conditions (i.e. when there is less memory available)

We are using this folder to develop this implementation, which is currently not suitable for production use.
Our plan is to move the packages in this folder to the top level of the repository once they are ready to be used.

```